tests/PHPSemVerChecker/Analyzer/ClassMethodAnalyzerTest.php

```diff
@@ -697,15 +697,46 @@ public function testClassMethodImplementationChanged($context, $visibility, $cod
 		$this->assertSame('tmp::tmpMethod', $report[$context][$expectedLevel][0]->getTarget());
 	}
 
-	public function providerImplementationChanged()
-	{
-		return [
-			['class', 'public', 'V023'],
-			['class', 'protected', 'V024'],
-			['class', 'private', 'V025'],
-			['trait', 'public', 'V052'],
-			['trait', 'protected', 'V053'],
-			['trait', 'private', 'V054'],
-		];
-	}
+    public function providerImplementationChanged()
+    {
+        return [
+            ['class', 'public', 'V023'],
+            ['class', 'protected', 'V024'],
+            ['class', 'private', 'V025'],
+            ['trait', 'public', 'V052'],
+            ['trait', 'protected', 'V053'],
+            ['trait', 'private', 'V054'],
+        ];
+    }
+
+    public function testClassMethodCaseChangeIsIgnored()
```

Comment:
This needs to be tested against classes, traits and interfaces, at each visibility (public, protected, private). Use a provider similar to `providerImplementationChanged` to accomplish this.

Reply:
The method name shouldn't be `testClassMethodCaseChangeIsIgnored` but `testClassMethodNameCaseChanged`.